pkg/collector/corechecks/oracle-dbm/activity_test.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build oracle
+//go:build oracle_test
```

Comment:
I believe this causes these tests to not get run at all, because `oracle_test` is not a build tag we recognize. Right now, I'm seeing the following in CI jobs ([example](https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/289675752)):
```
∅  pkg/collector/corechecks/oracle-dbm
∅  pkg/collector/corechecks/oracle-dbm/common
∅  pkg/collector/corechecks/oracle-dbm/config
```

If these tests are meant to be run with `invoke test`, I think you would get the expected result (ie. this code is tested, except on Heroku), by specifying:
```suggestion
//go:build oracle && test
```
(`test` is a build tag we add when running unit tests, so specifying both this and `oracle` allows this code to be run only on tests, and not when testing the Heroku flavor where the `oracle` tag is not set).

Reply:
Actually, the intention of `oracle_test` is prevent running tests, because they are not integrated in CI yet. For now, I'm running the tests locally.